TestVectorGenerator/Complex.cs

```diff
@@ -77,6 +77,7 @@ public void Generate(Stream outputStream)
 			document.UsageRules.Add(new UsageRule
 			{
 				KeyId = document.ContentKeys.First().Id,
+				IntendedTrackType = "TrackType1",
```

Comment:
Should test vectors reflect real-world-like data, i.e. have UHD etc. as IntendedTrackType?

Reply:
Yes, maybe better this way.